ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -43,10 +43,13 @@ public class GltfModelExporter
         public IFileLoader FileLoader { get; set; }
         public bool ExportMaterials { get; set; } = true;
         public bool AdaptTextures { get; set; } = true;
+        public bool SatelliteImages { get; set; } = true;
 
         private string DstDir;
         private CancellationToken? CancellationToken;
 
+        private readonly byte[] tempPng = new byte[] { 137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82, 0, 0, 0, 1, 0, 0, 0, 1, 8, 6, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 73, 69, 78, 68, 174, 66, 96, 130 };
```

Comment:
Add a comment explaining why this is required

Reply:
Done.